src/app/services/locales/en-us/index.yaml

```diff
@@ -9,3 +9,5 @@ services:
         service-config-view: 'YAML'
     items:
       title: Services
+  detail:
+    no_matching_external_service: 'No matching ExternalService was found for service {name}'
```

Comment:
Can we add this inside of `routes.item`? I'd probably just name it like a `not_found` or something seeing as eventually it will be the backend giving us this instead of a frontend 'match'.

Reply:
If we put everything into `routes.item` why is `routes.` in there to begin with? Seems superfluous. And then can’t everything be on `detail` for detail views?

Reply:
because we should also have `components.` (and maybe more top levels)

Reply:
Why should we have `components.`? Message keys shouldn’t be grouped by arbitrary technical details but by purpose.

Reply:
Doesn't matter, lets move this on.

Reply:
Line 11 of src/app/services/views/ServiceConfigView.vue refers to `services.routes.item.navigation.service-config-view` is makes sense at least to me that the key to the i18n 5 lines later would be routed in the same area (.i.e: `services.routes.item`